MainBrain/src/Model/Unitek/Unitek.hpp

```diff
@@ -29,8 +29,8 @@ struct ErrorBits
     uint8_t RaceAway        : 1;
     uint8_t User            : 1;
     uint8_t I2R             : 1;
-    uint8_t Reserve         : 1;
-    uint8_t HwFail          : 1;
+    uint8_t HwFail         : 1;
+    uint8_t AdcInt          : 1;
```

Comment:
just want to make sure these are the correct names

This is where got the original names from:
![image](https://user-images.githubusercontent.com/4438744/37739702-5dabaefa-2d31-11e8-9534-a3315a59fa2b.png)

Reply:
Yea there is a table in the NDrive manual (pg. 18) that has the changes I made. I went with that table because the new function accounts for error in both so if either one is correct it will at least HW_FAIL and reserve should stay low if it exists.

Reply:
Okay just wanted to make sure haha yeah that sounds good in that case